realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1492,6 +1493,11 @@ public Long call() throws Exception {
 
                         return handoverTableViewPointer;
 
+                    } catch (BadVersionException e) {
+                        // In some rare race conditions, this can happen. In that case, just ignore the error.
+                        RealmLog.d("findAllAsync handover could not complete due to a BadVersionException. " +
```

Comment:
What should people do with this message? Be concerned? Change anything in their code? Report it?

Do anyone know what a "BadVersionException" mean? It sounds really bad... :-)
What does "Retry SHOULD be" mean? Should it be "Retry WILL be" or "Retry IS"?